docs/guides/es2k/running-infrap4d.md

```diff
@@ -0,0 +1,281 @@
+# Run Infrap4d on Intel IPU E2100
```

Comment:
`infrap4d` (change from uppercase `I` to lowercase `i`)

Reply:
Same comment throughout the document

Reply:
Derek suggested following this on my other PR:

There is no universal style for capitalization of titles and headings.

A sampling of Markdown-specific alternatives:

GitHub Markdown Heading Capitalization Congruence: Our rough convention is that titles use "Title Case" and headers use "Sentence case". ... no periods. However, we haven't consistently applied this so far.
Mesos Markdown Style: Capitalize the first word of any heading, including title, subtitle, subheading. Capitalize all "major" words (nouns, verbs, adjectives, adverbs, and pronouns) in any heading, including the second part of hyphenated major words (e.g., Self-Report not Self-report).
Google Developer Markdown Style Guide: In document titles and headings, use sentence case. That is, capitalize only the first word in the title, the first word in a subheading after a colon, and any proper nouns or other terms that are always capitalized a certain way.
Our developers are wildly inconsistent in how they capitalize headings. I've seen headings at the same level bounce back and forth between title and sentence case in the same document. I've also seen random capitalizations in title case.

I'm proposing that we use Title case for levels 1 and 2 (# and ##) and Sentence case for levels 3 and below. This distinguishes between "major" and "minor" divisions, or between "headings" and "subheadings". It seems like a reasonable tradeoff.

Reply:
I think in general that makes sense (using capital letters for headings/subheadings/titles, but this is a program that we are specically calling "infrap4d" and not "Infrap4d" -- which Derek also pointed during my previous documentation notes. We should stick to consistent naming